### PR TITLE
Some more sd-bus docs

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: man/sd_bus_message_get_signature.xml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    </funcsynopsis>
+
+  </refsynopsisdiv>
+
+  <refsect1>
+    <title>Description</title>
+
+    <para><function>sd_bus_message_get_signature()</function> returns the signature of message
+    <parameter>message</parameter>. If <parameter>complete</parameter> is true, the signature of the
+    whole message is returned, and just the signature of the currently open container otherwise.
```
Comment (yuwata) [inline man/sd_bus_message_get_signature.xml:54]: 'opened'?

File: man/sd_bus_message_read_array.xml [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    </funcsynopsis>
+  </refsynopsisdiv>
+
+  <refsect1>
+    <title>Description</title>
+
+    <para><function>sd_bus_message_read_array()</function> gives access to an element array in
+    message <parameter>m</parameter>. The "read pointer" in the message must be right before an
+    array of type <parameter>type</parameter>. As a special case, <parameter>type</parameter> may be
+    <constant>NUL</constant>, in which case any type is acceptable. A pointer to the array data is
```
Comment (yuwata) [inline man/sd_bus_message_read_array.xml:44]: 'NULL'

File: man/sd_bus_message_get_signature.xml (hunk 1 above)
Comment (keszybz) [reply man/sd_bus_message_get_signature.xml:54]: "Opened" would imply that there's a transition and the state is just being changed to open. "Open" implies that the state is ongoing, which is more accurate. So I think it is OK as is.

File: man/sd_bus_message_read_array.xml (hunk 2 above)
Comment (keszybz) [reply man/sd_bus_message_read_array.xml:44]: It's a byte, so NUL (that's how it's called in old ASCII docs...). We use NUL fairly consistently for this in the tree.

**Comment by yuwata** [comment]
Thank you for the clarification.

**Pull request merged**